core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -148,6 +148,10 @@ public HubResponse<Enrollee> enroll(PortalParticipantUser operator,
         if (!studyEnvConfig.isAcceptingEnrollment()) {
             throw new IllegalArgumentException("study %s is not accepting enrollment".formatted(studyShortcode));
         }
+        if (preEnrollResponseId == null && hasRequiredPreEnroll(studyEnv.getId())) {
+            throw new IllegalArgumentException("user did not complete required pre-enrollment survey");
```

Comment:
FYI: this will be a change for OurHealth.  We may want to notify them or mark their preEnroll as not required

Reply:
Their preEnroll is already not required; this is true of all existing studies AFAIK